pkg/controller/util/selectors/bimultimap.go

```diff
@@ -152,7 +152,7 @@ func (m *BiMultimap) Put(key Key, labels map[string]string) {
 		m.selectingByLabeled[labelsKey] = selecting
 	}
 	selecting := m.selectingByLabeled[labelsKey]
-	selecting.refCount += 1
```

Comment:
The original code is easier for a reviewer to understand: you don't have to remember whether Golang's [`++` statement](https://go.dev/ref/spec#IncDec_statements) has side effects. In some similar languages, `--` and `++` are easy-to-misuse operators, not statements. However `+=` and `-=` are pretty much the same everywhere.

Do we have guidelines for Kubernetes about which style we prefer?

Reply:
All right, respect what you mean, I closed it